apps/frontend/src/app/sessions/select-play-level/client-page.tsx

```diff
@@ -5,17 +5,16 @@ import { useMemo } from "react"
 
 import { BackNavigationBar } from "@/components/Composite/BackNavigationBar"
 import { ExpandedSessionCard } from "@/components/Composite/booking/sessions/ExpandedSessionCard"
-import { Button } from "@/components/Generic/ui/button"
-import { useToast } from "@/components/Generic/ui/use-toast"
 import { useBookingMutation } from "@/hooks/mutations/booking"
 import { useCartStore } from "@/stores/useCartStore"
 import { PlayLevel } from "@/types/types"
+import { Box, Button, Container, Flex, For, VStack, useNotice } from "@yamada-ui/react"
 
 export default function ClientSelectPlayLevelPage() {
   const router = useRouter()
 
   const cart = useCartStore((state) => state.cart)
-  const { toast } = useToast()
+  const notice = useNotice()
 
```

Comment:
I think it's better to have the repeated config here.

```suggestion
  const notice = useNotice({
    placement: "bottom-right",
    isClosable: true,
  })

```